Decred Wallet/Extensions/UIColor.swift

```diff
@@ -10,27 +10,43 @@ import Foundation
 import UIKit
 
 extension UIColor {
-    // todo: significantly fewer colors should be left here once the implementation of the new
-    // interface design is completed as the new design uses fewer colors generally.
     struct appColors {
-        // decred colors are from https://decred.org/brand/
-        static let decredBlue = UIColor.init(hex: "#2970FF")
-        static let decredGreen = UIColor.init(hex: "#41BF53")
-        static let decredOrange = UIColor.init(hex: "#ED6D47")
-        
         static let darkYellowWarning = UIColor.init(hex: "#E7C659")
-        static let green = UIColor.init(hex: "#2DD8A3")
-        static let lightGray = UIColor.init(hex: "#DEE1E3")
-        static let lighterGray = UIColor.init(hex: "#C4CBD2")
-        static let offWhite = UIColor(hex:"#F3F5F6")
-        static let lightOffWhite = UIColor(hex: "#F9FBFA")
-        static let yellowWarning = UIColor.init(hex: "#FFC84E")
+        static let _lightGray = UIColor.init(hex: "#DEE1E3") // temporary name, pending when it will be made obsolete and removed
```

Comment:
I just want to confirm that it can be removed and use `gray` (#e6eaed) instead. 
`_ligthGray` is used on `Overview Page` wrongly instead of `gray`.
It is also used on `RecoverExistingWallet Page`, but that page will be replaced by the new UI.

Reply:
Makes sense.